lib/Ix/DBIC/ResultSet.pm

```diff
@@ -254,7 +250,7 @@ sub ix_get_updates ($self, $ctx, $arg = {}) {
 
     my @return = $ctx->result($res_type => {
       oldState => "$since",
-      newState => ($hasMoreUpdates
+      newState => "" . ($hasMoreUpdates
```

Comment:
I think the stringification should go into ix_highest_state. It's already in ix_state_string